testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -151,6 +151,13 @@ def test_toggle_atom_nomatch_mismatch_atoms(self, universe, reference):
         with pytest.raises(SelectionError):
             align.alignto(u, ref, select='all', match_atoms=False)
 
+    def test_no_atom_masses(self, universe, reference,
```

Comment:
Your test is [failing](https://travis-ci.com/github/MDAnalysis/mdanalysis/jobs/298558678#L2264) since your AtomGroup does contain masses. You should create an empty one without masses.

Reply:
Just to elaborate on what @PicoCentauri said -- any Universe you read from a file will have masses, so you should create your own from `mda.Universe.empty`.